Source/Controllers/DataControllers/SPDatabaseStructure.m

```diff
@@ -517,8 +524,6 @@ - (void)_cloneConnectionFromConnection:(SPMySQLConnection *)aConnection
 		// If a connection is already set, ensure it's idle before releasing it
 		if (mySQLConnection) {
 			[self _cancelAllThreadsAndWait];
-
-			(void)([mySQLConnection autorelease]), mySQLConnection = nil; // note: aConnection could be == mySQLConnection
```

Comment:
Should connection still be set to nil?

Reply:
Not necessary, line below there is a setter to the variable